docs/reference/api/docker-io_api.md

```diff
@@ -10,502 +10,5 @@ parent = "smn_remoteapi"
 
 # Docker Hub API
 
-- This is the REST API for [Docker Hub](https://hub.docker.com).
-- Authorization is done with basic auth over SSL
-- Not all commands require authentication, only those noted as such.
+This API is deprecated as of 1.7. You should use the [Docker Registry HTTP API V2](https://docs.docker.com/registry/spec/api/).
```

Comment:
I'm fine with removing this. This API isn't for end-user consumption. And of all the service integrations using this API, I'd say:
* If they already has the integration working, it'll continue to work. And they don't need the API reference.
* If they're planning on writing new integration, they shouldn't use the v1 API. I know that there's no replacement. But I also don't care much about these service integrators right now. We should be focusing more on end users.